tests/test_db.py

```diff
@@ -5966,6 +6146,7 @@ def _noop(*args, **kwargs):
 
         versions_migrate_tests_dict = {
             # This version removes the releases_history table, which is verified by other tests
+            33: self._add_release_json_tables,
```

Comment:
Nit, the new test should go above the comment, so the latter stays attached to the `32: _noop,`.